src/server.ts

```diff
@@ -27,12 +26,13 @@ import config from './config'
 import lookupTableRoutes from './lookup-table/routes'
 import snapshotRoutes from './components/snapshots/routes'
 import { routeLocalisations, routeGetLanguageFile } from './components/localisation/routes'
+import { extractJWTfromHeader, getTokenData } from './components/permissions/loginHelpers'
 require('dotenv').config()
 
 // Fastify server
 
 const startServer = async () => {
-  await loadActionPlugins() // Connects to Database and listens for Triggers
+  // await loadActionPlugins() // Connects to Database and listens for Triggers
```

Comment:
@andreievg -- I assume this was just so it would load quicker and you intended to put it back?